source/FreeRTOS_ARP.c

```diff
@@ -925,7 +925,7 @@ static BaseType_t prvFindCacheEntry( const MACAddress_t * pxMACAddress,
                  pxEndPoint != NULL;
                  pxEndPoint = FreeRTOS_NextEndPoint( NULL, pxEndPoint ) )
             {
-                if( ENDPOINT_IS_IPv4( pxEndPoint ) )
+                if( ( pxEndPoint )->bits.bIPv6 == 0U ) /*NULL End Point is checked in the for loop, no need for an extra check */
```

Comment:
```suggestion
                if( pxEndPoint->bits.bIPv6 == 0U ) /*NULL End Point is checked in the for loop, no need for an extra check */
```
Please remove redundant bracket.

Reply:
updated